types/recharts/index.d.ts

```diff
@@ -714,6 +715,13 @@ export interface XPadding {
 	left: number;
 	right: number;
 }
+export interface XLabel {
+	value: string;
+	position?: string;
+	offset?: number;
+	stroke?: string;
+	fontSize?: number;
```

Comment:
Are stroke and fontSize SVG or CSS attributes? If so, the interface Label should extend from the corresponding SVG interface resp. `Partial<CSSStyleDeclaration>` instead of defining the property itself.